make/config.mk

```diff
@@ -105,6 +105,12 @@ USE_LAPACK = 1
 # path to lapack library in case of a non-standard installation
 USE_LAPACK_PATH =
 
+# by default, disable lapack when using MKL
```

Comment:
OK, but if I want to build MXNet with full MKL and lapack, then I can still override this, right?
It is just the default for "normal" people

Reply:
yes